src/Network/Receive.pm

```diff
@@ -8889,11 +8889,17 @@ sub skill_msg {
 sub msg_string {
 	my ($self, $args) = @_;
 
-	if ($msgTable[++$args->{index}]) { # show message from msgstringtable.txt
-		message "$msgTable[$args->{index}]. Value: $args->{paral}\n", "info";
+	my $index = ++$args->{index};
+	if ($msgTable[$index]) { # show message from msgstringtable.txt
+		message "$msgTable[$index]. Value: $args->{paral}\n", "info";
 	} else {
-		warning TF("Unknown msgid:%d paral:%d. Need to update the file msgstringtable.txt (from data.grf)\n", $args->{index}, $args->{paral});
+		warning TF("Unknown msgid:%d paral:%d. Need to update the file msgstringtable.txt (from data.grf)\n", $index, $args->{paral});
 	}
+
+	Plugins::callHook('packet_msgstring', {
+		index => $index,
+		val => $args->{para1}
```

Comment:
paraL, not para1

Reply:
https://github.com/OpenKore/openkore/blob/master/src/Network/Receive/ServerType0.pm#L479
we use `para1`

Reply:
so need to fix it in #8896

Reply:
what is "paral"?

Reply:
that #8896 doesn't exist. Is it a typo @alisonrag ?

Reply:
line 8896* ^^
ya4ept already fixd this